ui/src/authorizations/apis/index.ts

```diff
@@ -17,3 +17,16 @@ export const createAuthorization = async (
     throw error
   }
 }
+
+export const getAuth0Config = async (): Promise<Auth0Config> => {
+  try {
+    const {data} = await AJAX({
```

Comment:
why are you doubling down on using the old request library for new features?

https://giphy.com/embed/AuIvUrZpzBl04

Reply:
That's a great question. I did a global search of influxdb for `/api/v2private` and followed that convention since we aren't actually generating a route for this particular endpoint on IDPE

Reply:
use fetch then